gems/blockchain_wallet/CVE-2019-15224.yml

```
@@ -0,0 +1,16 @@
+---
+gem: blockchain_wallet
+cve: 2019-15224
+ghsa: 333g-rpr4-7hxq
+url: https://github.com/rubygems.org/issues/2097
+date: 2019-08-20
+title: Critical severity vulnerability that affects blockchain_wallet
+description: |-
+  The blockchain_wallet gem 0.0.6 through 0.0.7 for Ruby, as distributed on RubyGems.org, included a code-execution backdoor inserted by a third party.
+  Users of an affected version should consider downgrading to the last non-affected version of 0.0.5.
+unaffected_versions:
+  - "< 0.0.6"
+  - "> 0.0.7"
+related:
+  url: 
+  - https://github.com/rubygems.org/wiki/Gems-yanked-and-accounts-locked#19-aug-2019
```

I believe this URL is incorrect and should be: 

```suggestion
  - https://github.com/rubygems/rubygems.org/wiki/Gems-yanked-and-accounts-locked#19-aug-2019
```

This incorrect URL is repeated through out the PR.